docs/book/src/introduction/sway_quickstart.md

```diff
@@ -202,6 +202,34 @@ Note the contract ID — you will need it if you want to build out a frontend to
 
 If `forc-deploy` emits a `no such subcommand` error, try to install it with [fuelup](../introduction/installation.md) or [manually](https://github.com/FuelLabs/sway/releases).
 
+## Deploying the contract with Salt
+
+You can now pass a --salt flag in your forc deploy command to redeploy a contract that has been deployed already. This is useful for multiple deployments of the same contract.
+
+In order to use the `--salt` flag, install the latest `forc` update by running the following command in your project directory:
+
+```bash
+cargo add forc
+```
+
+Alternately, you can add the following to your `cargo.toml` file:
+
+```toml
+forc = "0.35.2"
+```
```

Comment:
Hmm I have a couple of small concerns here:

If we follow the quickstart we can see that step by step we are describing creating a new project with `forc new` which creates a basic sway project that _does not_ contain a `Cargo.toml` by default. Executing `cargo add forc` would fail on such project as it is not a rust project that we can use cargo to add a dependency. 

Also I don't think we should ever depend on `forc` in a `Cargo.toml` as `forc` is a binary that you need to install. If you want to install `forc` via cargo, you can use `cargo install forc`. 

This opens up the next point: unless we have a reason not to, we should use `fuelup` for the installation process. For example installing newest `forc` with `cargo install forc` would not install `forc-client` which is the package that enables `forc deploy` and `forc run`. You have to manually install `forc-client` with `cargo install forc-client` to be able to use them. Installing via `fuelup` takes care of this automatically and installs both of them. So we may want to point them to update via `fuelup`. Or for some reason we need to point them to installation with cargo we need to mention `cargo install forc-client`.